UNO/UserInterface/HowToPlayPanel.java

```diff
@@ -1,5 +1,138 @@
 package UserInterface;
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Desktop;
+import java.awt.Dimension;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
 
-public class HowToPlayPanel {
+import javax.swing.*;
 
+public class HowToPlayPanel extends JPanel {
+
+    private Driver driver;
+    
+    public HowToPlayPanel (Driver driver)
+    
+    {
+      
+      this.driver = driver;
+      
+      JButton rulesButton = DesignUtils.createSpecialButton("Rules");
+      JButton returnButton = DesignUtils.createButton("Return to Start Menu");
+      
+      
+      // Load the UNO logo
+      ImageIcon icon = DesignUtils.loadUnoLogo(1000, 200);
+      
+      // Set preferred size to 1000x800
+      setPreferredSize(new Dimension(1000, 800));
+
+      // Set BorderLayout for the HowToPlayPanel
+      setLayout(new BorderLayout());
+
+      // Create a JLabel for the title
+      JLabel rulesTitleLabel = DesignUtils.createTitleLabel("How to Play UNO");
+      
+      // ActionListeners
+      
+      returnButton.addActionListener(e -> driver.showPanel("StartUpPanel"));
+      
+      rulesButton.addActionListener(new ActionListener()
+      {
+      
+      @Override
+      public void actionPerformed(ActionEvent e)
+      
+      {
+        //Source for accessing a URL--https://stackoverflow.com/questions/10037644/opening-a-url-from-a-jbutton-in-simple-java-program
+        try
+        {
+          URI uri = new URI("https://service.mattel.com/instruction_sheets/42001pr.pdf");
```

Comment:
Our rules are slightly different. Like player count, how points are initially set.

So I would incorporate a disabled for clicking textArea that has a scroll wheel with all of our updated rules. You can copy and paste what is in the website and use it, but tweak and add to it to match the rules in our requirements and design document.